app/server/auth/strategies/twitter.js

```diff
@@ -0,0 +1,12 @@
+import { Strategy } from 'passport-twitter';
+
+const TWITTER_CONSUMER_KEY = process.env.TWITTER_CONSUMER_KEY || 'key';
```

Comment:
It may equally be simplified to:

```javascript
const consumerKey = process.env.TWITTER_CONSUMER_KEY || 'key';
const consumerSecret = process.env.TWITTER_CONSUMER_SECRET || 'secret';
const callbackURL = 'http://localhost:8080/auth/twitter/callback';

const config = { consumerKey, consumerSecret, callbackURL };
const callback = (token, tokenSecret, profile, cb) => cb(null, profile);

export const twitter = new Strategy(config, callback);
```